packages/asset/contracts/Asset.sol

```diff
@@ -40,6 +41,7 @@ contract Asset is
     ERC2771HandlerUpgradeable,
     ERC1155BurnableUpgradeable,
     AccessControlUpgradeable,
+    OwnableUpgradeable,
```

Comment:
Did you check that the order of inheritance didn't affected the storage slots ?

Reply:
The oz upgrades plugins does that (it used to be a little bit buggy and didn't worked with solc < 0.5.13) we must consider using it but it won´t be easy because we need to decide if we keep using `hardhat-deploy` plugin or not.